filament/src/materials/MaterialHelpersShading.fxh

```diff
@@ -530,4 +656,4 @@ void ApplyAllPostPrepare(inout MaterialInputs material, inout FragmentData fragm
 
     ApplyNonTextured(material, fragmentData);
     ApplyShaprScalars(material, fragmentData);
-}
\ No newline at end of file
+}
```

Comment:
nit: missing newline

Reply:
Weird, locally, I have a newline but I've added an extra one in c680e4f2bf6838ae0f7480a18558640779362379

Reply:
I think it depends on the editor you open it with. I trust HxD on Windows, or Hex Fiend on macOS 🔡

Reply:
It's the opposite, @gaborvalasek's changes in 9959d5a3cb7adc0309d68b951171fb8858ee894c added the newline which had been missing from master. (Remember, this is how GitHub shows this, the red icons signals **missing** newline).
<img width="172" alt="image" src="https://user-images.githubusercontent.com/53563131/217195602-0b5d2306-1825-4fc7-8eed-979c1d6c3aa2.png">